enterprise/app/usage/usage.tsx

```diff
@@ -84,8 +84,14 @@ export default class UsageComponent extends React.Component<UsageProps, State> {
                   <div className="usage-value" title={formatWithCommas(period.totalDownloadSizeBytes)}>
                     {formatBytes(period.totalDownloadSizeBytes)}
                   </div>
+                  <div className="usage-resource-name">Total bytes uploaded from cache</div>
+                  <div className="usage-value" title={formatWithCommas(period.totalUploadSizeBytes)}>
+                    {formatBytes(period.totalUploadSizeBytes)}
+                  </div>
                   <div className="usage-resource-name">Linux remote execution duration</div>
                   <div className="usage-value">{formatMinutes(Number(period.linuxExecutionDurationUsec))}</div>
+                  <div className="usage-resource-name">Saved CPU Time</div>
```

Comment:
Not sure we want to show these in the UI yet until they're populated for a month or so. Maybe you can leave them in here but comment them out?